Dockerfile

```diff
@@ -3,8 +3,8 @@
 #
 # 🎯 Version Management
 #
-ARG CORRETTO_VERSION="17-alpine3.19"
-ARG CORRETTO_SHA="2122cb140fa94053abce343fb854d24f4c62ba3c1ac701882dce12980396b477"
+ARG CORRETTO_VERSION="21-alpine3.17"
```

Comment:
why a downgrade of alpine versione?
let's update to 3.20 instead:
https://hub.docker.com/layers/library/amazoncorretto/21-alpine3.20/images/sha256-f3e9cef8e99e55bae620641a79fd4cdf361c6911862c0fa304072d88e6739472?context=explore